randovania/games/prime1/json_data/Phazon Mines.txt

```diff
@@ -88,6 +88,8 @@ Extra - aabb: [58.95912, -13.365204, -0.8746805, 213.88028, 134.1634, 99.340744]
       Any of the following:
           Space Jump Boots
           Morph Ball Bomb and Morph Ball and Bomb Jump (Beginner)
+          Grapple Beam and Movement (Beginner)
+          Boost Ball and Morph Ball and Wall Boost (Expert)
```

Comment:
I think there's also a slope jump required to the wallboost, no?

Reply:
I'm not sure, there wasn't a video along with it. Do you know of one? I thought Wall boost wasn't enough but i took your word for it.